.circleci/config.yml

```
@@ -155,13 +183,21 @@ jobs:
             adb shell settings put global transition_animation_scale 0.0
             adb shell settings put global animator_duration_scale 0.0
       - *adb_logging
-      - *build_test_android
+      - run:
+          name: Run android unit test
+          command: ./gradlew connectedDebugAndroidTest --debug --stacktrace
       - store_test_results:
           path: android_instrumented_test/build/outputs/androidTest-results/connected
-      - persist_to_workspace:
-          root: ~/
-          paths: 
-            - circleci-caver-java-android/*
+      - store_artifacts:
+          path: ~/circleci-caver-java-android/
+          destination: adb.log
+
+  build_test_android:
+    <<: *machine_ubuntu
```

```suggestion
    <<: *android_machine_ubuntu
```
Because this is a build test for android, we must use android machine here.

---

@aeharvlee applied